posthog/tasks/usage_report.py

```diff
@@ -284,8 +284,14 @@ def capture_event(
     name: str,
     organization_id: str,
     properties: Dict[str, Any],
-    timestamp: Optional[datetime] = None,
+    timestamp: Optional[Union[datetime, str]] = None,
 ) -> None:
+    if timestamp and isinstance(timestamp, str):
+        try:
+            timestamp = parser.isoparse(timestamp)
```

Comment:
are you sure this timestamp is always an iso string?

Reply:
I think it should be. But I just added a commit to make sure it is.